cmd/clean.go

```diff
@@ -226,16 +227,13 @@ func executeClean(region string) error {
 			numVerDeleted := countDeleteVersions(globalLambdaDeleteInputStructs)
 			log.Info(fmt.Sprintf("%d unique versions will be removed in an actual execution.", numVerDeleted))
 			spaceRemovedPreview := calculateSpaceRemoval(globalLambdaDeleteList)
-			log.Info(fmt.Sprintf("%s of storage space will be removed in an actual execution.", humanize.Bytes(uint64(spaceRemovedPreview))))
-
-		}
-
-		if !DryRun {
+			log.Info(fmt.Sprintf("%s of storage space will be removed in an actual execution.", humanize.IBytes(uint64(spaceRemovedPreview))))
+		} else {
```

Comment:
We don't really need the else. Let's just end the original `if DryRun {}` block and allow the rest of the code to execute. That's cleaner.

Reply:
Modified the format of this now but had to move the duration calculation and output to a function to remove code duplication.